.github/workflows/backport.yml

```diff
@@ -31,5 +31,5 @@ jobs:
           pull_description: |-
             Bot-based backport to `${target_branch}`, triggered by a label in #${pull_number}.
 
-            * [ ] Before merging, ensure that this backport complies with the [Criteria for Backporting](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#criteria-for-backporting-changes).
+            * [ ] Before merging, ensure that this backport complies with the [Criteria for Backporting](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#changes-acceptable-for-releases).
```

Comment:
maybe the wording here could actually match the subheader
```
ensure that this backport is [acceptable for release]
```

Reply:
ie. dropping the criteria word would drop the expectation of explicit checklist.

Reply:
Nice idea, did that!